ouroboros-network/src/Ouroboros/Network/Diffusion/Configuration.hs

```diff
@@ -87,14 +95,28 @@ defaultPraosTargets =
 -- | These targets are established when Genesis mode is enabled
 -- in node configuration and when the node is syncing up
 --
-defaultGenesisSyncTargets :: PeerSelectionTargets
-defaultGenesisSyncTargets =
-  defaultPraosTargets {
+defaultSyncTargets :: PeerSelectionTargets
+defaultSyncTargets =
+  defaultDeadlineTargets {
     targetNumberOfActivePeers               = 0,
     targetNumberOfKnownBigLedgerPeers       = 100,
     targetNumberOfEstablishedBigLedgerPeers = 50,
     targetNumberOfActiveBigLedgerPeers      = 30 }
```

Comment:
We should reduce the other targets.  Every hot outbound connection is costly both in terms of memory and CPU.  In `defaultDeadlingTargets` we have `20` active peers all together, here we have `45`.  I'd advocate for `targetNumberOfActivePeers = 0`, which will be a 50% increase in active peers where all of them are big ledger peers.

Reply:
The big ledger peers target is separate from normal peers so the default config has 25 active peers.

The cost of hot peers is due to chainsync, and consensus is (has?) addressing it with "chain-skip".
In the discussions I've been a part of consensus have said that they need 50 hot connections to big ledger peers to maintain security guarantees. We should test the performance with 50 hot peers to see if "chain-skip" improves things enough.